weaver/processes/wps_package.py

```diff
@@ -962,8 +962,8 @@ def _handler(self, request, response):
 
             self.update_status("Launching package...", PACKAGE_PROGRESS_LAUNCHING, STATUS_RUNNING)
 
-            self.is_ems = get_weaver_configuration(self.settings) == WEAVER_CONFIGURATION_EMS
-            if self.is_ems:
+            self.remote_execution = get_weaver_configuration(self.settings) in WEAVER_CONFIGURATION_HYBRID
```

Comment:
Not sure about the `in WEAVER_CONFIGURATION_HYBRID` ? Do you mean `in WEAVER_CONFIGURATIONS_REMOTE` ?